src/main/java/seedu/apollo/ui/Parser.java

```diff
@@ -143,10 +144,10 @@ private static Command parseCommand(String[] split, int size, ModuleList moduleD
             return new ShowModuleCommand(split[1], moduleData);
 
         case COMMAND_LIST_MODULE_WORD:
-            if (!isOneWord(split)) {
-                throw new IllegalCommandException();
+            if (isOneWord(split)) {
+                return new ListModuleCommand();
             }
-            return new ListModuleCommand();
+            return new ListModuleWithLessonCommand(split[1], moduleData);
```

Comment:
This way will still work if user does smth like `listmod cs2113 hello`, nth to change just to confirm that we are doing this

Reply:
Are we allowing `listmod cs2113 hello`?

Reply:
I don't think we should allow listmod cs2113 with trailing invalid arguments